Docs/Getting Started Guide.md

```diff
@@ -279,10 +270,10 @@ store.subscribe(self) { subcription in
 
 ### Asynchronous Operations
 
-Conceptually asynchronous operations can simply be treated as state updates that occur at a later point in time. Here's a simple example of how to tie an asynchronous network request to `ReSwift` state update:
+Conceptually asynchronous operations can simply be treated as state updates that occur at a later point in time. Here's a simple example of how to tie an asynchronous network request to a `ReSwift` state update:
 
 ```swift
-func fetchGitHubRepositories(state: State, store: Store<State>) -> Action? {
+func fetchGitHubRepositories(_ state: State, _ store: Store<State>) -> Action? {
```

Comment:
Aside from the reducer signatures, what's your motivation to remove parameter names here?